configs/tx2-sst.yaml

```diff
@@ -0,0 +1,185 @@
+---
```

Comment:
Would it be better to put all sst related config files in the sst folder itself?